scale/scheduler/models.py

```diff
@@ -10,27 +12,36 @@
 
 
 class SchedulerManager(models.Manager):
-    '''Provides additional methods for handling scheduler db entry
-    '''
+    """Provides additional methods for handling scheduler db entry
+    """
 
     def get_master(self):
-        '''Gets the current master scheduler instance for the cluster.
+        """Gets the current master scheduler instance for the cluster.
 
         :returns: The master scheduler.
         :rtype: :class:`scheduler.models.Scheduler`
-        '''
+        """
         try:
             return Scheduler.objects.get(pk=1)
         except Scheduler.DoesNotExist:
             logger.exception('Initial database import missing master scheduler: 1')
             raise
 
+    def initialize_scheduler(self):
```

Comment:
Based on the PR note about the scheduler fixture being removed in order to add a DB field that was also later removed...should we go back to just having the fixture or is there still a compelling reason to use this custom initialization code?